coop_membership/data/email_template_data.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<odoo noupdate="1">
+<odoo noupdate="0">
```

Comment:
Don't change it.
noupdate should remain True

Reply:
@ivantodorovich :done